build.gradle

```diff
@@ -24,6 +25,14 @@ repositories {
 	maven { url 'https://repo.spring.io/snapshot' }
 }
 
+jar{
```

Comment:
indentation is too much from line 28 to 34 
please reformat it

Reply:
reformatted